stack.yaml

```diff
@@ -2,3 +2,6 @@ resolver: lts-19.1
 
 ghc-options:
   "$locals": -fwrite-ide-info
+
+extra-deps:
+  - persistent-2.14.0.0
```

Comment:
this is how i tested, can definitely back out

Reply:
IIUC, `persistent-2.14` has landed in `nightly` (we got a notification about it breaking us), so the `stack-nightly.yaml` file should be a good test (and is run on CI).